### DISCUSSION TITLE
Plotting second time series?

### DISCUSSION THREAD

**Discussion by trophyhead** [kernc/backtesting.py#999] [General]
Is it possible to plot a second timeseries with the Backtest.plot() function?  Other than the OHLC data being operated upon.

For example, if trading decisions are made based on Heiken-Ashi plots or another alternative candlestick, could this be plotted as well (much as the indicators are)?

**Comment by einarjohnson** [comment]
wondering about the same thing, did you ever find an elegant solution to this?
